src/vs/workbench/contrib/preferences/browser/settingsEditor2.ts

```diff
@@ -1210,10 +1243,47 @@ export class SettingsEditor2 extends EditorPane {
 			this.hasWarnedMissingSettings = true;
 		}
 
-		const commonlyUsed = resolveSettingsTree(commonlyUsedData, dividedGroups.core, this.logService);
-		resolvedSettingsRoot.children!.unshift(commonlyUsed.tree);
+		const toggleData = await getExperimentalExtensionToggleData(this.workbenchAssignmentService, this.environmentService, this.productService);
+		if (toggleData) {
+			for (const key in toggleData.settingsEditorRecommendedExtensions) {
+				const prerelease = toggleData.settingsEditorRecommendedExtensions[key].onSettingsEditorOpen!.prerelease;
+				const groupTitle = toggleData.settingsEditorRecommendedExtensions[key].onSettingsEditorOpen!.groupTitle;
```

Comment:
You can get the group title by getting the manifest of the extension as follows:

```ts
const manifest = await this.extensionGalleryService.getManifest(extension, CancellationToken.None);
if (!manifest) {
	continue;
}
groupTitle = manifest.contributes.configuration.title;
```